### PR TITLE
 Fix issues on Manage ratings feature 

### PR DESCRIPTION
**Pull request by Dev-nkundineza**
### PR Description
This PR resolve the issues on manage ratings
### Description of tasks that were expected to be completed
- [x] Manager can see all ratings
- [x] On updated remarks, there is no need for admin approval
- [x] The coordinator has to get an email if there is a rejection/ approval of the  ratings 
List all the tasks and sub-tasks that were assigned to you or that you assigned yourself. Make sure you check every completed task.
### Functionality
How do this works, add some information that may be needed in testing
### How has this been tested?

- [ ]  Clone frontend repository [frontend](https://github.com/atlp-rwanda/atlp-pulse-fn.git)
- [ ] Clone backend repository [](https://github.com/atlp-rwanda/atlp-pulse-bn.git)
- [ ] checkout the branch   fix-manage-ratings on both frontend and backend 
- [ ] Add the ratings to a trainee
- [ ] Update the Ratings
- [ ] Login as admin, view all ratings

### CODE REVIEW

**Review by alicepretty — APPROVED** [review]

**Review by Procech20 — COMMENTED** [review]
Can you check the reason why your review app is not being created?!

**Comment by Procech20** [comment]
I deployed a review app for you to try out this pull request 👇🏽

Heroku App: https://devpulse-bn-review-app-44.herokuapp.com/


<hidden purpose="for-rewritable-pr-comment-action-use" value="Deploy"></hidden>

File: src/helpers/emailForTrainee.helper.ts [hunk 1]
```diff
@@ -41,9 +41,9 @@ export default function generateTemplate() {
                                     <tr>
                                         <td style="padding:0 35px;">
                                             <h1 style="color:#1e1e2d; font-weight:500; margin:0;font-size:32px;font-family:'Rubik',sans-serif;">
-                                            Dear Trainee,
+                                            ${title}
     
-                                            This is to inform that , new devpulse ratings are out now
+                                            ${content}
```
Comment (Procech20) [inline src/helpers/emailForTrainee.helper.ts:46]: Nice job for making the code re-usablde

**Review by Procech20 — APPROVED** [review]

**Review by PatrickTUy — APPROVED** [review]
Approved

**Review by ceelogre — APPROVED** [review]

**Pull request merged**